src/lib/PhpCsFixer/Config.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformCodeStyle\PhpCsFixer;
+
+use PhpCsFixer\Config as ConfigBase;
+
+class Config extends ConfigBase
+{
+    public const EZPLATFORM_PHP_HEADER = <<<'EOF'
```

Comment:
As I've said internally, it might be too much for general use. I'd prefer an opt-in (in a form of a method?).
However not gonna block this PR because of that.

Reply:
I tackled this in: https://github.com/ezsystems/ezplatform-code-style/pull/3/commits/3325b117d107f97f44dfa4b36acc44ee70c7459c Let me know if you are okay with `EzPlatformInternalConfigFactory::build`. It uses our `Config` instance and reconfigures it to provide our header comment and possibly can accommodate more specific rules in the future.